src/atomizer.js

```diff
@@ -77,7 +77,7 @@ var GRAMMAR = {
     'SIGN'       : 'neg',
     'NUMBER'     : '[0-9]+(?:\\.[0-9]+)?',
     'UNIT'       : '[a-zA-Z%]+',
-    'HEX'        : '[0-9a-f]{3}(?:[0-9a-f]{3})?',
+    'HEX'        : '[0-9A-F]{3}(?:[0-9A-F]{3})?',
```

Comment:
I'm not a big fan of uppercase hex values (feels like something I'd never do in CSS).  One way to avoid collisions is to just use 6 chars instead of 3 in these cases.  Or, we could introduce hex syntax that is more explicit (eg, `C-#333`), which would help us when dealing with numeric custom suffixes too (eg, 'C-100' should be valid as a custom suffix, and not be misinterpreted for a hex value, but today it's not possible.)